src/modules/75-cf/components/DetailPageTemplate/DetailPageTemplate.module.scss

```
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+dt::after {
+  content: ': ';
+}
+
+.metaData {
+  display: flex;
+  position: absolute;
+  top: var(--spacing-xxlarge);
+  right: var(--spacing-xxlarge);
+  margin-right: 190px;
+
+  dt {
+    font-weight: bold;
+  }
```

You might as well move this in.
```suggestion
.metaData {
  display: flex;
  position: absolute;
  top: var(--spacing-xxlarge);
  right: var(--spacing-xxlarge);
  margin-right: 190px;

  dt {
    font-weight: bold;
    
    &::after {
      content: ': ';
    }
  }
```

---

moved